diff --git a/drivers/spi/spidev.c b/drivers/spi/spidev.c
index 859910ec8d9f..d4cb765d9c8d 100644
--- a/drivers/spi/spidev.c
+++ b/drivers/spi/spidev.c
@@ -682,6 +682,11 @@ static const struct of_device_id spidev_dt_ids[] = {
 	{ .compatible = "lwn,bk4" },
 	{ .compatible = "dh,dhcom-board" },
 	{ .compatible = "menlo,m53cpld" },
+	{ .compatible = "ti,lmk04832" },
+	{ .compatible = "ti,lmk04208" },
+	{ .compatible = "ti,lmk04828b" },
+	{ .compatible = "ti,lmx2594" },
+	{ .compatible = "nxp,sc18is602b"},
 	{},
 };
 MODULE_DEVICE_TABLE(of, spidev_dt_ids);
